### DISCUSSION TITLE
Not getting verification mail when using thorugh GitPod

### DISCUSSION THREAD

**Discussion by sumitbishti** [documenso/documenso#832] [General]
I tried using documenso through GitPod but i am not getting verification mail !!

**Comment by Mythie** [comment]
Hey there, when using Documenso through Gitpod you'll find the mailserver running on port 9000 just like when performing local development.

If you navigate to the ports tab you will be able to see the currently open ports with links to access the services running on them:

<img width="1097" alt="image" src="https://github.com/documenso/documenso/assets/13398220/db09d2d5-4c96-4d87-a886-7be28a4f1140">

**Comment by sumitbishti** [comment]
Got it. Thanks now working.
